projects/canopy/src/styles/mixins.scss

```diff
@@ -33,39 +33,34 @@
   $focus-color: var(--link-focus-color),
   $focus-bg-color: var(--link-focus-bg-color)
 ) {
-  $box-shadow-inset-width: -0.063rem;
-
-  border-bottom: 0.125rem solid $default-color;
   color: $default-color;
   line-height: initial;
-  padding: 0 0.125rem;
-  text-decoration: none;
+  text-decoration: underline;
```

Comment:
I think if we change this mixin, we will have to change the link itself too, otherwise we'll have discrepancies in how buttons styled as links and links will look.

If Alex agrees to change the pattern, we should also add:
```
    text-decoration-thickness: 0.125rem;
    text-underline-offset: 0.125rem;
```

Reply:
Links will be changed and will look the same as they use the same mix in.
https://github.com/Legal-and-General/canopy/blob/5dd34a8bc74cd6cd92e8248b40f6f9b215381050/projects/canopy/src/styles/mixins.scss#L280
I've glanced in Storybook over the elements that use this mix in and I've not seen visible changes.

Reply:
There is quite a lot of difference in how the links/buttons styled as links look now, even if we don't extend the sides, we should at least make them look more like the design: https://percy.io/ffaa5218/web/LG-Canopy/builds/43112747

I think those property should do the job. Might need to look at the other states though.